Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -91,59 +91,58 @@ @interface Branch() <BNCDebugConnectionDelegate, UIGestureRecognizerDelegate, BN
 
 @implementation Branch
 
-static Branch *currInstance;
-
 #pragma mark - Public methods
 
 
 #pragma mark - GetInstance methods
 
 + (Branch *)getInstance {
-    return [Branch getBranchInstance:YES];
+    NSString *branchKey = [BNCPreferenceHelper getBranchKey:YES];
+    if (!branchKey || [branchKey isEqualToString:NO_STRING_VALUE]) {
```

Comment:
Hmm, that's strange. If you look at line 1003 in the deleted diff, we weren't actually checking there either. We probably should here, though.